demos/golang/grpc_pingpong/run_ping_on_occlum.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -e
+
+BLUE='\033[1;34m'
+NC='\033[0m'
+
+occlum_ping="./occlum_ping"
+
+if [ ! -f $occlum_ping ];then
+    echo "Error: cannot stat file '$occlum_ping'"
+    echo "Please see README and build it using prepare_ping_pong.sh"
+    exit 1
+fi
+
+# Init Occlum Workspace
+rm -rf occlum_ping_context && mkdir occlum_ping_context
```

Comment:
We have renamed `occlum context` to `occlum instance`,  use `occlum_ping_instance` as the name